### PR TITLE
Added 2 new themes

### PR DESCRIPTION
**Pull request by jdorfman**
Trying out a new theme affiliate that pays 30% per sale.

### CODE REVIEW

**Comment by XhmikosR** [comment]
re `npm i` leftover. Do `npm audit fix` and it'll be like in the repo.

File: config/_extras.yml [hunk 1]
```diff
@@ -12,6 +12,12 @@ books:
     img: /assets/img/books/Bootstrap-Hours-Sams-Teach-Yourself-ebook.jpg
     url: 'https://www.amazon.com/Bootstrap-Hours-Sams-Teach-Yourself-ebook/dp/B017M8OWZQ/ref=as_li_ss_il?ie=UTF8&linkCode=li3&tag=bcdn-20&linkId=f8a39275b5204f5f2d0427a1ef1c4f83'
 themesAd:
+  - name: 'Velonic - Admin Dashboard & Frontend'
```
Comment (XhmikosR) [inline config/_extras.yml]: I'll fix this later, tomorrow top, must be escaped thus the failing test.

**Comment by jdorfman** [comment]
@XhmikosR 

> Do `npm audit fix` and it'll be like in the repo.

Can this be added to the `README`?

ty

**Comment by XhmikosR** [comment]
No, because IMO it's a bug so this is just a workaround...

**Pull request merged**